megamek/src/megamek/common/WeaponOrderHandler.java

```diff
@@ -233,8 +233,8 @@ public synchronized static void saveWeaponOrderFile() throws IOException {
                 weapOrderMap.put(unitId, weapOrder);
             }
             return weapOrderMap;
-        } catch (Exception e) {
-            throw new IOException(e);
+        } catch (Exception ex) {
+            throw new IOException(ex);
```

Comment:
I gotta say that the idea of re-throwing a generic exception as an IOException is a little weird, but we probably better leave it as is for now.

Reply:
I do too, but wanted to keep the rethrows consistent for now (I am planning on looking at them once everything is migrated to proper logging methods).